controllers/operands/kubevirt_test.go

```diff
@@ -1271,6 +1285,42 @@ Version: 1.2.3`)
 					})
 				})
 
+				Context("should ignore SRIOVLiveMigration on SNO ", func() {
+					BeforeEach(func() {
+						hcoutil.GetClusterInfo = func() hcoutil.ClusterInfo {
+							return &commonTestUtils.ClusterInfoSNOMock{}
+						}
+					})
```

Comment:
Do we want to restore `hcoutil.GetClusterInfo` after the test?

Reply:
it should be covered by the AfterEach on the external context (line 1227)